rclcpp_components/src/component_manager.cpp

```diff
@@ -160,6 +160,18 @@ ComponentManager::create_node_options(const std::shared_ptr<LoadNode::Request> r
                 "Extra component argument 'use_intra_process_comms' must be a boolean");
       }
       options.use_intra_process_comms(extra_argument.get_value<bool>());
+    } else if (extra_argument.get_name() == "use_global_arguments") {
```

Comment:
@gezp meta: thinking about your use case, I wonder if it would make sense for the `component_manager` node to expose, e.g. as a parameter, the default value of `use_global_arguments` for the nodes it composes. Seems convenient if you're already passing a large parameter file to it.

Reply:
i agree with you, for my use case (use a large parameter file for all components), it seems better to use a parameter of  `component_manager` to configure all components `use_global_arguments`, which we needn't set this for components one by one in launch file. 
I'm not good at naming parameter, could you give me some suggestions ? @hidmic . how about  `use_global_arguments`  (i think it's bad, because it seems that we set `component_manager` nodes to use global arguments, but we want to set this for components)  or `component_use_global_arguments`?

Reply:
@gezp perhaps `forward_global_arguments`? I'm not very creative either. At any rate, I won't block this patch on it.